### DISCUSSION TITLE
平板不配活着吗

### DISCUSSION THREAD

**Discussion by cat333333cat3** [openai/whisper#1966] [General]
平板不配活着吗

**Comment by onlyjokers** [comment]
?

**Comment by cat333333cat3** [comment]
平板用不了
「ziqi ***@***.***>」在 2024年1月20日 週六，02:05 寫道：

**Reply by onlyjokers** [reply]
通过[https://github.com/ggerganov/whisper.cpp](url)该项目，应该是能在平板上运行较小的模型的

**Comment by Mattral** [comment]
What is that, a riddle?

**Comment by cat333333cat3** [comment]
This is a popular saying in China nowadays, meaning why can't this one
work, I don't deserve it?

「Min Htet Myet (Mattral) ***@***.***>」在 2024年1月20日 週六，12:47
寫道：

**Reply by Mattral** [reply]
Ah I get it now thank you!

**Comment by cat333333cat3** [comment]
Can you guys have this software in the mall, as a matter of fact there is
one in the mall, but it doesn't work when transcribing, it automatically
ignores words that are very close to the tone of voice, or converts the
tone of voice that shouldn't be converted to text. And it's an old model,
can you guys earn them out one, and can you pay with Chinese payment
methods, like Alipay.

「Min Htet Myet (Mattral) ***@***.***>」在 2024年1月20日 週六，12:47
寫道：

**Reply by Mattral** [reply]
Whisper may struggle to accurately transcribe speech if there is a lot of background noise. If there are specific words or phrases that are consistently being misinterpreted, try providing additional context or emphasizing those words to improve accuracy.

**Comment by cat333333cat3** [comment]
我不满足，我要转录三小时的音频
「ziqi ***@***.***>」在 2024年1月20日 週六，12:54 寫道：

**Reply by onlyjokers** [reply]
3小时的转译我没有尝试过，但是我尝试用这个去转译半小时的项目，可以正常的运行。
也许可以试试通过ffmpeg来把3小时的音频给变成多个半小时的音频？3小时的转译我没有尝试过，但是我尝试用这个去转译半小时的项目，可以正常的运行。
也许可以试试通过ffmpeg来把3小时的音频给变成多个半小时的音频？

**Comment by cat333333cat3** [comment]
拜托你们在商城上架这个音频转文字软件，跪求了，一个月199 我都买我求求去了

「冯炎炎 ***@***.***>」在 2024年1月20日 週六，12:55 寫道：

**Comment by cat333333cat3** [comment]
求求你们努力一下 把它训练成3个小时的王者 199元一个月的费用那种 我相信很多人会因为他的功能买单 之所以199
因为我是个穷鬼超过这个价格我就买不起了
「ziqi ***@***.***>」在 2024年1月20日 週六，13:01 寫道：

**Reply by onlyjokers** [reply]
openai 官方有这个api可以调用的：[https://platform.openai.com/docs/api-reference/audio/createTranscription](https://platform.openai.com/docs/api-reference/audio/createTranscription)，国内应该有很多这种中转的服务，我自己是在新加坡部署了一个服务器用作数据中转，它可以工作的很好
或者微软的 Azure 服务，甚至连中转服务都不需要，在国内可以直接访问

**Comment by cat333333cat3** [comment]
谢谢各位看得起我 要是我可以 我有你们这样的技术 我就不会在这里无能狂怒了

「ziqi ***@***.***>」在 2024年1月20日 週六，13:09 寫道：
